presto-native-execution/CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ if(PRESTO_ENABLE_REMOTE_FUNCTIONS)
 endif()
 
 set(VELOX_BUILD_TESTING
-    OFF
```

Comment:
This is not desired since it will increase the build time.
I fixed the build issue here https://github.com/facebookincubator/velox/pull/8206